### DISCUSSION TITLE
Disable initial password when starting the app

### DISCUSSION THREAD

**Discussion by pablolerner** [2fast-team/2fast#120] [Ideas]
Hello. I am trying 2fast and it looks like a great app and it works perfectly, thanks very much for building it!
I wanted to ask if it's possible to remove asking for a password every time I open the app. I am currently trying it out as a possible replacement of Authy desktop, which does not ask me for a password every time I open the app and it feels a more streamline workflow. I understand that it's an extra layer of security but at the same time it feels annoying to be entering a password every time I just want a TOTP code to access some other web site or application. At the same time my personal position is that I trust my PC enough that I feel this is not necessary. Not everybody will feel the same, I understand.
Is it currently possible to configure the app like this? If not, would it be possible to add this functionality?

**Comment by jp-weber** [comment]
Thank you.
The password is mandatory in the process, as the data file is encrypted with it. If i were to change this process with an option, this would considerably reduce security. 

However, there is the possibility of using Windows Hello for login. A simple pin could be assigned in Windows Hello to make the login easier.

**Reply by pablolerner** [reply]
And what about the Windows Credential Manager? Could it be used to store the encryption key when the data file is created? And maybe automatically decrypt the data file with the same key, reading it from the Credential Manager when needed.

**Reply by jp-weber** [reply]
In fact, the Windows Credential Manager ist already in use for Windows Hello, but I don't know how configurate the option without to create a bypass and destroy the security. The creation without password and the data file in plain text would be a "clean solution"

**Reply by pablolerner** [reply]
Thanks for the answer
